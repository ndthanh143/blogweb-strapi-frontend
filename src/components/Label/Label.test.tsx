import React from 'react';
import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import { Label } from '.';

describe('Label component', () => {
  test('should render correctly', () => {
    const component = renderer.create(<Label>You are the best!</Label>);
    let tree = component.toJSON() as ReactTestRendererJSON;
    expect(tree).toMatchSnapshot();
  });
});
