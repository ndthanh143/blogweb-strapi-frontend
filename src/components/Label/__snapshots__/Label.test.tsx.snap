// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component should render correctly 1`] = `
<span
  className="px-4 py-1 rounded-lg font-medium bg-label-light text-color-primary dark:bg-label-dark hover:bg-color-primary dark:hover:bg-color-primary hover:text-white"
>
  You are the best!
</span>
`;
